eslint.config.js

```diff
@@ -1,5 +1,167 @@
 import blumilkDefault from '@blumilksoftware/eslint-config'
+import stylisticPlugin from '@stylistic/eslint-plugin'
+import importPlugin from 'eslint-plugin-import'
+
+const custom = {
+  rules: {
+    'function-multiline-arguments': {
+      meta: {
+        type: 'layout',
+        fixable: 'code',
+        schema: []
+      },
+      create: context => ({
+        CallExpression: node => {
```

Comment:
It looks like new custom rule, maybe create this rule in different file, and just import here. It will be much better to read this config.